sagemaker-jumpstart/object_detection/JumpStart_Object_Detection.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        entry_point="transfer_learning.py",
</pre><p>Where is this script coming from?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>It is in source_dir. Added a comment mentioning the same.</p>